packages/cdktf-cli/bin/cmds/ui/models/terraform-cloud.ts

```diff
@@ -320,4 +343,26 @@ export class TerraformCloud implements Terraform {
     await Promise.race([ready(), timeout()]);
     logger.debug('Configuration Version is ready in Terraform Cloud');
   }
+
+  public outputParser(line: string): DeployingResource | undefined {
+    let message: CloudLog;
+    try {
+      message = JSON.parse(line) as CloudLog
+    } catch (error) {
+      throw new Error(`Could not parse Terraform Cloud output: ${line}`);
+    }
+
+    const resourceMatch = message.hook?.resource?.resource;
+    const applyState = getResourceState(message['@message']); // TODO: use type instead
+
+    if (applyState && resourceMatch && resourceMatch.length >= 0 && resourceMatch[1] != "Warning") {
```

Comment:
I imagine the `Warning` check isn't needed in this case.

Reply:
Yeah, we get this safely from the hook now, should not be needed anymore